### PR TITLE
sshguard role to enable whitelisting

### PR DESCRIPTION
**Pull request by thraxil**
Installs sshguard (it comes default on GCP machines, but we need it for AWS/Azure). Sets up the whitelist with entries from `sshguard_whitelist` variable.

### CODE REVIEW

**Comment by OmarIthawi** [comment]
That's awesome! Thank you @thraxil! Will take a look as soon as possible.

**Comment by thraxil** [comment]
OK, apparently just starting the systemd service makes the role non-idempotent (despite the ansible docs claiming otherwise). Going to look into a workaround for that.

**Comment by thraxil** [comment]
[InterSystems: Automate the IPs whitelisting for SSH Guard](https://trello.com/c/sGtj7eXp/3076-intersystems-automate-the-ips-whitelisting-for-ssh-guard)

File: sshguard/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  apt:
+    pkg: sshguard
+    state: present
+    update_cache: yes
+  tags: ['sshguard', 'sshguard:install']
+
+- name: update sshguard whitelist
+  lineinfile:
+    dest: /etc/sshguard/whitelist
+    regexp: "^{{item}}"
```
Comment (OmarIthawi) [inline sshguard/tasks/main.yml]: `{{item}}` -> `{{ item }}`, with spaces just like most of our playbooks' convention.

**Comment by OmarIthawi** [comment]
@thraxil the role looks great! I've added a few followup notes (usual scope creep :see_no_evil:) in the Trello card to make sure that we actually benefit from this role.

File: sshguard/tasks/main.yml (hunk 1 above)
Comment (thraxil) [reply sshguard/tasks/main.yml]: Fixed. Thanks!

**Comment by OmarIthawi** [comment]
Thanks @thraxil! I'd say we should wait for the postmortem to be concluded and solidified before we merge this PR.

My hope that we'll only need this role for customers who are accessing the servers via SSH (or SFTP). Currently I think there's only ISC.

**Pull request merged**